chat/src/test/java/net/md_5/bungee/api/chat/ComponentsTest.java

```diff
@@ -20,6 +20,19 @@ public void testItemParse()
         String serialised = ComponentSerializer.toString( component );
         BaseComponent[] deserialised = ComponentSerializer.parse( serialised );
         Assert.assertEquals( TextComponent.toLegacyText( deserialised ), TextComponent.toLegacyText( component ) );
+        //////////
+        TextComponent component1 = new TextComponent( "HoverableText" );
+        String nbt = "{display:{Name:{text:Hello},Lore:[{text:Line_1},{text:Line_2}]},ench:[{id:49,lvl:5}],Unbreakable:1}}";
```

Comment:
Why the hell using an int for ID. Enchantments are defined as NamespacedKeys/string